### DISCUSSION TITLE
search_domain_filter, sub-domain and increase limit

### DISCUSSION THREAD

**Discussion by damianoneill** [perplexityai/api-discussion#33] [Ideas]
Today, the current solution is limited to only 3 domains for whitelisting and blacklisting.  It would be great if this limit could be increased to 10. 

In line with increasing the limit, it would be useful to be able to limit based on sub-domains. For example, I imagine a lot of API users limit based on company content, typically hosted under xyz.com. Having the ability to limit based on xyz.com/content-a, xyz.com/content-c/sub-content, and xyz.com/content-e/sub-content/more-content would be very beneficial.

**Comment by mikehudson2** [comment]
Yes, please can we have sub-domains. Really important....

**Comment by shubhang98** [comment]
We are looking into adding sub domains. Will have updates on this soon.

**Comment by Eochs** [comment]
Seconding this, but I would like to see the limit increase to over 100. Many parents whitelist a set of sites for their kids (e.g. Khan Academy and other edtech platforms) and having a search interface default to searching and surfacing content only in those safe sites would solve a major pain point.
